portfolio/src/main/webapp/script.js

```diff
@@ -202,6 +202,126 @@ function deleteComments(myLocation){
 
 }
 
+/* Google Maps integration - Week 4 Libraries project */
```

Comment:
Maybe split code for different features into different files so that this file won't grow too big.